modules/pensions/spec/models/pensions/saved_claim_spec.rb

```diff
@@ -51,7 +51,7 @@
 
     describe '#process_attachments!' do
       it 'sets the attachments saved_claim_id' do
-        expect(Lighthouse::SubmitBenefitsIntakeClaim).to receive(:perform_async).with(claim.id)
+        expect(Lighthouse::SubmitBenefitsIntakeClaim).not_to receive(:perform_async).with(claim.id)
```

Comment:
pensions is not using this job - can/should this check be removed?

Reply:
I altered these two tests because in the OG saved_claim it calls `SubmitBenefitsIntakeClaim` so my thought was to now make sure it's NOT being hit with the override in place. It's definitely not a hill I need to die on so I'll defer to you.